src/common/storage/backends/s3/metadata.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "S3 Storage",
+  "configStructure": [
+    {
+      "name": "endpoint",
+      "displayName": "S3 Server URL",
+      "description": "URL for S3 Server",
+      "value": "",
+      "valueType": "string",
+      "readOnly": false
+    },
+    {
+      "name": "accessKeyId",
+      "displayName": "Access Key",
+      "description": "Access Key for S3 Server.",
```

Comment:
Not sure if we need a description for the next couple - they don't really add any meaningful information. Unless we wanted to provide examples.

Reply:
The description for Secret Key and Access Key has been removed.